### ISSUE TITLE
Scale the lr when using different samping num

### ISSUE THREAD

**Issue by lxa9867** [facebookresearch/Mask2Former#23]
Hi,

Thank you for sharing such a good work! I have a question regarding to the loss. I found if I use more frames during training, the loss goes very high. Do I need to scale down the lr linearly according to the sampling frame num? Thank you.

**Comment by bowenc0221** [comment]
I played with video instance segmentation out of curiosity and thus have not done thorough ablations on using a different number of frames during training (and currently I'm not planning to do more experiments on VIS). I'm afraid you will need to ablate the choice of learning rate yourself if you are interested.

**Comment by lxa9867** [comment]
Thank you for your reply！
